src/ProjectNeon/Assets/Scripts/Saver.cs

```diff
@@ -0,0 +1,33 @@
+﻿using UnityEngine;
+
+
+[CreateAssetMenu(fileName = "New Saver", menuName = "Saver")]
+public class Saver : ScriptableObject
```

Comment:
Let's rename this to `Settings`. We will only be saving settings from the Options menu.